Source/Orts.Simulation/Simulation/Timetables/TTTrain.cs

```diff
@@ -13298,6 +13520,7 @@ public bool PerformDetach(TTTrain train, bool allowPlayerSelect)
                 Trace.TraceInformation("Train {0} : detach to train {1} : cannot find new train \n", train.Name, DetachFormedTrainName);
             }
 
+//            newTrain.Autopilot = train.Autopilot;
```

Comment:
I see there are a couple of lines of functional code that are commented out, if you don't see any use for these lines anymore they might as well be removed to improve readability.

Reply:
Done